microsoft-edge/webview2/release-notes/index.md

```diff
@@ -218,8 +220,10 @@ The following APIs have been promoted to Stable and are now included in this Rel
 ---
 
 
-<!-- ------------------------------ -->
-* Added `SaveAs` APIs that allow you to programmatically perform the **Save as** operation.  You can use these APIs to block the default **Save as** dialog, and then either save silently, or build your own UI for **Save as**.  These APIs pertain only to the **Save as** dialog, not the **Download** dialog, which continues to use the existing Download APIs.
+<!-- ---------- -->
+###### SaveAs
```

Comment:
```suggestion
###### SaveAs API
```
alts:
* SaveAs
* SaveAs API
* SaveAs API for custom Save As UX

@victorhuangwq

Reply:
result:
```
## 1.0.2739.15
#### Promotions
###### Web notification handling
###### Save as
```